src/osd/OSD.h

```diff
@@ -118,6 +118,7 @@ class OSDService : public Scrub::ScrubSchedListener {
 
   void enqueue_back(OpSchedulerItem&& qi);
   void enqueue_front(OpSchedulerItem&& qi);
+  double get_cost_per_io() const;
```

Comment:
This function isn't always valid -- it crashes the OSD if called on wpq.  That's ok here, but please warn the next reader by adding a comment:
```
/// scheduler cost per io, only valid for mclock, asserts for wpq
double get_cost_per_io() const;
```